core/modules/system/css/system.theme.css

```diff
@@ -112,10 +112,16 @@ input.form-checkbox,
 input.form-radio {
   vertical-align: middle;
 }
-.marker,
 .form-required {
   color: #f00;
 }
+.marker{
+  font-size:0.9em;
+  background-color:#DDD;
+  padding:0 0.4em;
+  -moz-border-radius: 3px;
+  border-radius: 3px;
+}
```

Comment:
Thanks @Graham-72! Could we format our CSS white-space consistently? There should be a space after each colon and before opening squiggly brackets. It also looks like colors in Seven are specified using lower-case letters.

See https://api.backdropcms.org/css-standards for the full docs on code style.